timemachine/cpp/src/segmented_weighted_random_sampler.cu

```diff
@@ -99,28 +99,34 @@ void SegmentedWeightedRandomSampler<RealType>::sample_given_noise_device(
 
 template <typename RealType>
 std::vector<int>
-SegmentedWeightedRandomSampler<RealType>::sample_host(const std::vector<std::vector<RealType>> &probabilities) {
-    const int num_segments = static_cast<int>(probabilities.size());
+SegmentedWeightedRandomSampler<RealType>::sample_host(const std::vector<std::vector<RealType>> &weights) {
+    const int num_segments = static_cast<int>(weights.size());
```

Comment:
should [wrap_kernels.cpp](https://github.com/proteneer/timemachine/blob/4d8787dc3529842a4bd06b81c24e9596b38d0d94/timemachine/cpp/src/wrap_kernels.cpp#L221-L227) be updated to match naming convention in https://github.com/proteneer/timemachine/pull/1240#discussion_r1466679534 ?

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1240/commits/5911687fc113dbbf7727aea7810c514e2b5ab162